### ISSUE TITLE
Empty libraries folder after running `rails hotwire:install` causes deployments to fail

### ISSUE THREAD

**Issue by justin-robinson** [hotwired/hotwire-rails#14]
# Problem
Deployments fail on remote build systems after running `rails hotwire:install`
```
Errno::ENOENT: No such file or directory @ dir_initialize - /app/app/assets/javascripts/libraries
/app/vendor/ruby-2.7.2/lib/ruby/2.7.0/pathname.rb:443:in `open'
/app/vendor/ruby-2.7.2/lib/ruby/2.7.0/pathname.rb:443:in `foreach'
/app/vendor/ruby-2.7.2/lib/ruby/2.7.0/pathname.rb:443:in `children'
/app/vendor/bundle/ruby/2.7.0/gems/stimulus-rails-0.1.2/lib/stimulus/importmap_helper.rb:11:in `block in importmap_list_from'
/app/vendor/bundle/ruby/2.7.0/gems/stimulus-rails-0.1.2/lib/stimulus/importmap_helper.rb:7:in `each'
/app/vendor/bundle/ruby/2.7.0/gems/stimulus-rails-0.1.2/lib/stimulus/importmap_helper.rb:7:in `flat_map'
/app/vendor/bundle/ruby/2.7.0/gems/stimulus-rails-0.1.2/lib/stimulus/importmap_helper.rb:7:in `importmap_list_from'
/app/vendor/bundle/ruby/2.7.0/gems/stimulus-rails-0.1.2/lib/stimulus/importmap_helper.rb:3:in `importmap_list_with_stimulus_from'
/app/app/assets/javascripts/importmap.json.erb:3:in `call'
```

# Recreation
1. create a new rails app
2. follow the install in this wiki
3. deploy your app (I used Heroku)

# Cause
` /app/app/assets/javascripts/libraries` does not exist
because
git ignores it
because
it's an empty folder

# Solutions
1. Add .keep file in ` /app/app/assets/javascripts/libraries` (what I did)
2. Modify `importmap_list_with_stimulus_from` to silently fail when a folder doesn't exist
3. Provide a default `hello_library.js`. This could also be a good place to demonstrate how libs are designed to be implemented and used

<img width="326" alt="image" src="https://user-images.githubusercontent.com/3090489/104108322-79537d80-5278-11eb-9fc6-a359a1a13d9e.png">

**Comment by jumichot** [comment]
I ran into this too, should be fixed here https://github.com/hotwired/stimulus-rails/pull/16

**Comment by dhh** [comment]
Fixed in 0.1.3 ✌️
